controller/controller.go

```diff
@@ -144,14 +149,27 @@ func (c *controller) updateIngresses() error {
 						}
 					}
 
+					if stripPath, ok := ingress.Annotations[stripPathAnnotation]; ok {
+						if stripPath == "true" {
+							entry.StripPaths = true
+						} else if stripPath == "false" {
+							entry.StripPaths = false
+						} else {
+							log.Debugf("Skipping entry as it has an invalid strip path annotation: %v", stripPath)
+							skipped++
```

Comment:
Should we really skip it? Or would it be better to just assume the default value?